main/src/ca/uwaterloo/flix/language/phase/jvm/BytecodeInstructions.scala

```diff
@@ -38,6 +39,17 @@ object BytecodeInstructions {
 
     def visitVarInstruction(opcode: Int, v: Int): Unit =
       visitor.visitVarInsn(opcode, v)
+
+    def visitJumpInstruction(opcode: Int, label: Label): Unit =
+      visitor.visitJumpInsn(opcode, label)
+
+    def visitLabel(label: Label): Unit =
+      visitor.visitLabel(label)
+
+    def visitLoadConstantInstruction(v: Any): Unit =
```

Comment:
Why is this "untyped"? Will it not be better - esp. for latter to have typed versions?

Reply:
Yeah I agree. I am just still trying to decide what is the responsibility of Instructions vs. F since they are pretty close. I wasn't sure what even is the requirements on that can be given as an ldc instruction. I think it's okay for now? and it will be clearer what to do once there are more cases of usage

Reply:
OK